gotchas/dhis2-release-lifecycles.md

```diff
@@ -0,0 +1,91 @@
+# DHIS2 release lifecycles
+
+This is a walk through on the DHIS2  software releases and its lifecycles. Starting off we are gonna communicate some basic concepts in order to build a common language that we will use to better explain the release cycle.
```

Comment:
"gonna" is slang - please use "going to"  :)

Reply:
changed it :) thanks